### ISSUE TITLE
Add validation for ask us a question field

### ISSUE THREAD

**Issue by Jolken** [ArbakovMaxim/construction#1]
Ask us a questions fields are not validated. Empty strings or invalid phone number could be provided

https://github.com/ArbakovMaxim/construction/assets/22051865/287360b7-4317-4a70-b0f0-029846391b24

**Comment by ArbakovMaxim** [comment]
Indeed, not setting the limit to the minimum number of digits, and not checking using trim for the presence of simple spaces, is a childish mistake... thanks for the hint.
